### DISCUSSION TITLE
ジャンパの半田がつかなくなってしまいました。

### DISCUSSION THREAD

**Discussion by fairytail7** [Yowkees/keyball#187] [General]
トラックボール側のジャンパの半田付けがつかなくなってしまいました。
4箇所全てになります。過去の記事で一箇所ダメにしている人を参考にエナメル線を繋げる等などで対象できることを願っておりますが4箇所という数と
回路図の見方がわからずまた、導線の方法も詳細が分からない為投稿させていただきます。
調べれば分かるようなことを質問しているかもしれませんが、ご教示いただければ幸いです。
お忙しいところ大変恐縮ですが、よろしくお願いします
![image](https://user-images.githubusercontent.com/29137240/213964901-fb9f1b22-92eb-4778-bf0f-1f42ac7c39c3.jpg)

**Comment by fairytail7** [comment]
Keyball 39となります

**Comment by Yowkees** [comment]
状況了解しました。

下記の画像の同色の端子同士を接続（短絡）させてください。これでOLEDを復活させることができます。
エナメル線やゴム被覆銅線など何を使用しても結構です。

なお、この作業はTRSジャック、リセットスイッチ、12Pピンソケットと4Pピンソケットを半田付け後に行ってください。

導線の半田付け方法はウェブ検索等で調べていただければ幸いです。
お手数ですがご対応よろしくお願いいたします。

![KB39_fix](https://user-images.githubusercontent.com/84209772/213985911-097f24e5-4646-4865-b968-9c8ba47a0c52.jpg)

**Comment by fairytail7** [comment]
ありがとうございます。
頑張ってみます
